resolwe/process/tests/processes/python_test.py

```diff
@@ -59,8 +59,10 @@ class MyGroup:
         my_group = GroupField(MyGroup, label="My group")
 
     class Output:
-        my_output = StringField(label="My output")
+        string_output = StringField(label="My string output")
```

Comment:
Git history (and also a single commit) is much cleaner if you only change relevant code.

Reply:
Well I had to add a `list_string_output` field for the test, so I had to change this field to be consistent...